### DISCUSSION TITLE
Looking into HA Android App issues

### DISCUSSION THREAD

**Discussion by EmilyNerdGirl** [ESPresense/ESPresense#112] [Support]
Tagging @DTTerastar  as may want to check out this issue, trying to figure out why Beaconscope is working perfectly, but HA's Android app isn't showing up again; https://github.com/home-assistant/android/issues/1855

**Comment by DTTerastar** [comment]
I'll pull out my android phone and see if I can figure it out...

**Reply by EmilyNerdGirl** [reply]
May have been a pre-mature post as just saw someone post there was an exception happening in the HA App logs. 

BTW have you seen this? https://developers.google.com/android/exposure-notifications/ble-attenuation-procedure#periodic-reference-calibrations

Thought about trying, Google's official way to calibrate I believe?

**Reply by DTTerastar** [reply]
I have read that.  Its focused on the lower powered covid exposure tracking, but yeah it could be helpful.

**Comment by EmilyNerdGirl** [comment]
Looks like the Android issue is resolved but waiting for a code review :)   Other interesting note is this, https://github.com/home-assistant/android/issues/1855#issuecomment-962104992
Hoping that means we likely won't have the issues where BeaconScope gets killed after awhile on Android?

**Comment by EmilyNerdGirl** [comment]
FYI I sideloaded the latest beta release with the fix last night, and it seems to be not only working, but so far more stable than Beaconscope as HA wakes every 15mins to get around Android Doze. This may be the release Android users have been waiting for :)

**Reply by DTTerastar** [reply]
I'm mostly an iPhone kinda person, but I ordered a Galaxy 10e from Swappa so I actually have a recent vintage phone to test with.
